alf/algorithms/monet_algorithm.py

```diff
@@ -51,6 +51,7 @@ def __init__(self,
                  filters: Tuple[int],
                  nonskip_fc_layers: Tuple[int],
                  output_channels: int,
+                 instancenorm: bool = True,
```

Comment:
add an underscore  (``use_instance_norm``) in the name to be consistent with other arguments?

Reply:
> add an underscore (`use_instance_norm`) in the name to be consistent with other arguments?

updated the name